src/commands/local.ts

```diff
@@ -172,29 +171,6 @@ export async function startLocalEnvironment(options: GenezioLocalOptions) {
     const localConfPath = yamlProjectConfiguration.workspace?.backend 
         ? path.join(yamlProjectConfiguration.workspace.backend, "genezio.local.yaml") 
         : undefined 
-    let yamlLocalConfiguration = await getLocalConfiguration(localConfPath);
```

Comment:
You removed completely these lines and the SDK is not generated anymore because projectConfiguration.sdk is always undefined.